codewars8.py

```diff
@@ -1,38 +1,15 @@
 # https://www.codewars.com/kata/58b3c2bd917a5caec0000017/train/python
-# Just testing something out. I'm about to give up.
+# I gave up and used groupby.
 
 def sum_groups(arr):
+    from itertools import groupby
```

Comment:
It certainly works, but while importing libraries, you should define them outside of the function. So it should be more or less like this:
```
from x import y

def function():
```

Reply:
Thank you for this! Applying changes and creating a new pull request.